src/wallet/rpcwallet.cpp

```diff
@@ -1381,6 +1381,7 @@ UniValue listtransactions(const JSONRPCRequest& request)
                 RPCResult{
             "[\n"
             "  {\n"
+            "    \"involvesWatchonly\": xxx, (bool) Only returned if imported addresses were involved in transaction\n"
```

Comment:
Matches the description from the `listreceivedbylabel` [help](https://github.com/bitcoin/bitcoin/blob/master/src/wallet/rpcwallet.cpp#L1243).

Reply:
I'd rather pull this out into a static const RPCResult in a moveonly commit, since it is used in three places. And then only update that single place with the documentation. We do that in other places (like getmempoolentry)

Reply:
Agree.